core/domain/exp_services.py

```diff
@@ -1293,17 +1298,18 @@ def update_version_history(
                 ),
                 committer_ids=updated_committer_ids
             ))
-        updated_version_history_model.update_timestamps()
-        updated_version_history_model.put()
+        return updated_version_history_model
 
 
-def _save_exploration(
+def _compute_models_to_put(
```

Comment:
This name is a bit generic, maybe _compute_models_to_put_when_saving_new_exp_version() (or is that too long)?